File: Selenium/Activities/Activity2.java
package Example;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;

public class Activity2{

 public static void main(String[] args) {
		  
   WebDriver driver = new FirefoxDriver();
		    
   driver.get("https://training-support.net/webelements/login-form/");
		   
   System.out.println("Title is: "+driver.getTitle());
		    
   WebElement element1 = driver.findElement(By.id("username"));
   
   element1.sendKeys("admin");
   
   WebElement element2 = driver.findElement(By.id("password"));
   
   element2.sendKeys("password");
   
   WebElement element3 = driver.findElement(By.xpath("//button[text()='Submit']"));
   
   element3.click();
   
   System.out.println("Closing the browser");
   
   driver.quit();
}
}

File: Selenium/Activities/activity2.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
  
  driver.get("https://training-support.net/webelements/login-form")
  
  print("Title is: ", driver.title)

  element1 = driver.find_element(By.ID, "username")
  element1.send_keys("admin")

  element2 = driver.find_element(By.ID, "password")
  element2.send_keys("password")
  
  element3 = driver.find_element(By.XPATH, "//button[text()='Submit']")
  element3.click()

  print("Closing the browser")

  driver.quit()